Source/Core/Core/PowerPC/Interpreter/Interpreter_Integer.cpp

```diff
@@ -9,13 +9,13 @@
 #include "Common/Logging/Log.h"
 #include "Core/PowerPC/PowerPC.h"
 
-void Interpreter::Helper_UpdateCR0(u32 value)
+static void Helper_UpdateCR0(PowerPC::PowerPCState* ppcs, u32 value)
```

Comment:
This can just be made a member function of PowerPCState

Reply:
I think the idea was to keep PowerPCState separate from interpreter-specific code.

Reply:
We should probably decide on a policy for adding methods to PowerPCState, since some other PRs have similar candidates (e. g. UpdateCR1 and SetSR in #7889).

Reply:
It's not really interpreter-specific, since it doesn't actually operate on any interpreter state, but operates on the PowerPC state exclusively

Reply:
… not that there's any "interpreter state" left after #7896, except for m_end_block, which I intend to convert into a return value when I change the interpreter routine signature to include a pointer/reference to something that contains PowerPCState.